bundles/org.openhab.binding.lghombot/README.md

```diff
@@ -0,0 +1,95 @@
+# LG HomBot Binding
+
+The binding integrates hacked LG HomBot VR6260 based vacuum robots.
```

Comment:
You use the word 'hacked', 'modify', 'modding', 'modded' to describe the same thing, maybe we should choose a single way of spelling it out.